tests/authentication/test_authentication_blank_field.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from allure import description, step, title, feature
+from hamcrest import assert_that, is_, contains_string
+import re
+
+from framework.endpoints.authenticate_api import AuthenticateAPI
+from framework.tools.generators import generate_user_data
+
+timestamp_pattern = r"\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}"
+
+
+@feature("Authentication of user")
+class TestAuthentication:
+    @title("Test authentication, negative scenario")
+    @description(
+        "GIVEN user is registered"
+        "WHEN fields password, email are blank "
+        "THEN status HTTP CODE = 400"
+    )
+    @pytest.mark.parametrize("email, password, expected_status_code, expected_message_part", [
+        (" ", " ", 400, "Email is the mandatory attribute"),
+        (None, " ", 400, "Password is the mandatory attribute"),
+        (" ", None, 400, "Email is the mandatory attribute")
+    ], ids=["password_and_email_blank", "password_blank", "email_blank"])
+    def test_authentication_with_various_blank_inputs(self, email, password, expected_status_code, expected_message_part):
+        with step("Generation data for registration"):
+            data = generate_user_data(length_first_name=5, length_last_name=5, password_len=8)
+
+        with step("Registration new user"):
+            response = AuthenticateAPI().registration(body=data)
+            assert_that(response.status_code, is_(201), reason='Expected status code 201')
+
+        with step("Preparation data for authentication request"):
+            "Generated email/password if the parameter is None"
```

Comment:
it's clear from the code what's going on

Reply:
Fixed